.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file
+
+*           # @ownerUnlessOverridden
+.github/**  # @CICD owners
+docs/**     # @docOwners
+**/*.js     # @jsOwners
+**/*.ts     # @tsOwners
```

Comment:
We don't want to hardcode owners in the template file because technically anyone can use it to create a new repo. However, we should provide some common patterns for typical file paths and separate owners for our expected usecases.